infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -131,7 +127,7 @@
      * the primary sequencer. This means that any snapshot timestamp below this
      * actual threshold would abort due to NEW_SEQUENCER cause.
      */
-    private final Cache<String, Long> conflictToGlobalTailCache;
+    private final Cache<String, ConflictTuple> conflictToGlobalTailCache;
```

Comment:
please update description for conflictToGlobalTailCache... also maybe ConflictTuple does not convey the actual meaning, wouldn't it be more suitable something like CommitTuple, which refers to the address and the transaction ID of this committed operation...